example_bt2/btx_filter/callbacks.c

```diff
@@ -0,0 +1,59 @@
+#include <metababel/metababel.h>
+#include <stdbool.h>
+#include <assert.h>
+#include <stdio.h>
+#include <string.h>
+
+struct data_s {
+  const char *names_value;
+};
+
+typedef struct data_s data_t;
+
+void btx_initialize_usr_data(void *btx_handle, void **usr_data) {
+  *usr_data = calloc(1, sizeof(data_t));
+}
+
+void btx_read_params(void *btx_handle, void *usr_data, btx_params_t *usr_params) {
+  ((data_t *)usr_data)->names_value = usr_params->names;
+}
+
+void btx_finalize_usr_data(void *btx_handle, void *usr_data) {
+  data_t *data = (data_t *)usr_data;
+  free(data);
+}
+
+static void btx_filter_condition(void *btx_handle, void *usr_data, const char* event_class_name, bool *matched)
+{
+  data_t *data = (data_t *)usr_data;
+
+  int str_size = strlen(data->names_value);
+  char str[str_size];
+
+  strcpy(str, data->names_value);
+
+  bool ocurrence = false;
+
+  char *ptr = strtok(str, ",");
+  while(ptr != NULL && !ocurrence)
+  {
+    ocurrence = (strcmp(ptr,event_class_name) == 0);
+    ptr = strtok(NULL, ",");
+  }
+
+  // If the event_class_name is found in 'data->names_value', the associated callback is called,
+  // this resulting in the event being discared.
+  *matched = ocurrence;
+}
+
+static void btx_filter_callback(void *btx_handle, void *usr_data, const char* event_class_name)
+{
+
```

Comment:
I don't think we need an empty `{}` here...

Reply:
Good point !!!